src/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,82 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import {Sticky, StickyContainer} from "react-sticky";
+
+import Step from "./Step";
+
+import "./Stepper.less";
+
+const propTypes = {
+  className: PropTypes.string,
+  style: PropTypes.object,
+  steps: PropTypes.arrayOf(PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    description: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
`description: PropTypes.node,`
(since PropTypes.node is a superset of PropTypes.string)